instance/main.tf

```diff
@@ -36,6 +36,7 @@ resource "aws_instance" "instance" {
   }
 
   credit_specification {
+    count  = "${module.cpu_credits_available.cpu_credits_available}"
```

Comment:
this won't work, `count` only works on resources or data resources, but not on nested objects.

TBH I would first test if the `credit_specification` object works on all instance types, maybe it it's set to  `standard` is good enough. You can test that locally before merging.

Reply:
ok @iuriaranda I'll try it first

Reply:
@iuriaranda I've tried it and it is not complaining in my plan if I use a t2 for example

Reply:
and what about the apply? Don't be afraid to create an instance to test this 😛